.github/workflows/sycl-linux-build.yml

```diff
@@ -245,6 +245,12 @@ jobs:
         # TODO consider moving this to Dockerfile.
         export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
         cmake --build $GITHUB_WORKSPACE/build --target check-sycl-unittests
+    - name: check-sycl-unittests-preview
+      if: always() && !cancelled() && contains(inputs.changes, 'sycl')
+      run: |
+        # TODO consider moving this to Dockerfile.
+        export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+        cmake --build $GITHUB_WORKSPACE/build --target check-sycl-unittests-preview
```

Comment:
IMO, existing target should just check both.

Reply:
Done in https://github.com/intel/llvm/pull/19687/commits/22d66d0a7353b46abf9c4b0626be5bc29f41c435